README.md

```diff
@@ -2,53 +2,48 @@ This GitHub Action syncs your current issue branch to a remote Pantheon repo, us
 
 # Parameters
 
-- `PANTHEON_REPO_URL` [required]
-
-Remote GIT Repo URL (in pantheon)
-
-- `PANTHEON_SITE_ID` [required]
-
-Pantheon site id (or site.env in terminus e.g: domain)
-
-- `ACTION` [required]
-
-Action to run: create-multidev | delete-multidev | merge-to-dev
-
-- `BRANCH_NAME` [required]
-  
-The branch you want to send to Pantheon
-
-- `STRICT_BRANCH_NAME` [optional]
-
-Set to strict to enforce Jira ticket naming for branches (`none` by default)
-
+| `VALUE`              |                                                                                   |
+|----------------------|-----------------------------------------------------------------------------------|
+| `ACTION`             | Remote GIT Repo URL (in pantheon)                                                 |
+| `BRANCH_NAME`        | Name of the branch                                                                |
+| `STRICT_BRANCH_NAME` | Boolean, `strict` if branch needs to adhere to JIRA formatting (eg: `ABC-12-foo`) |
+| `PANTHEON_SITE_ID`   | UUID of the site (eg: `16c190c7-ee1f-4c27-80f9-cc403f8a64dd`)                     |
```

Comment:
Are we sure it is a uuid ?  Pantheon doc says "site.env in terminus e.g: domain"